src/d3d9/shaders/d3d9_find_depth_bias_factor.vert

```diff
@@ -0,0 +1,13 @@
+#version 450
+
+void main() {
+  vec2 coord = vec2(
+    float(gl_VertexIndex & 2),
+    float(gl_VertexIndex & 1) * 2.0f);
+
+  gl_Position = vec4(
+    -1.0f + coord.x,
+    -1.0f + 3.0f * coord.y,
+    gl_VertexIndex == 2 ? 1.0 : 0.0,
```

Comment:
Making a gradient?

Reply:
Yes. 

> For floating-point depth attachment, there is no single minimum resolvable
difference. In this case, the minimum resolvable difference for a given polygon is dependent on the
maximum exponent, e, in the **range of z values spanned by the primitive**. If n is the number of bits
in the floating-point mantissa, the minimum resolvable difference, r, for the given primitive is
defined as: 2^(e-n)

Emphasis mine. But because of that we can't really solve it with a simple factor. I went with the factor for a range from 0-1 as a best effort.

The top left vertex is supposed to be at depth 0 because that's what we read back for the depth bias.